### PR TITLE
fix: design inconsistencies

### PR DESCRIPTION
**Pull request by pedroCloudoki**

### CODE REVIEW

File: lib/components/CTACard/CTACard.tsx [hunk 1]
```diff
@@ -46,9 +46,13 @@ const CTACard: React.FC<CTACardProps> = ({
           justify="space-between"
         >
           <Grid item xs={8}>
-            <CardContent>
+            <CardContent
+              style={{
+                padding: '24px 0px 24px 40px',
```
Comment (a31859) [inline lib/components/CTACard/CTACard.tsx:51]: use theme spacing

File: lib/pages/PublisherDetails/PublisherDetails.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ import { PUBLISHER_APPS_PER_PAGE } from '../../constants/globals'
 const PublisherDetails: React.FC = () => {
   const classes = useStyles()
 
-  const { palette } = useTheme()
+  const theme = useTheme()
```
Comment (a31859) [inline lib/pages/PublisherDetails/PublisherDetails.tsx:30]:
```suggestion
  const { palette, spacing } = useTheme()
```

File: lib/components/CTACard/CTACard.tsx [hunk 3]
```diff
@@ -60,8 +64,8 @@ const CTACard: React.FC<CTACardProps> = ({
           </Grid>
 
           <Grid item xs={4}>
-            <CardActions>
-              <Box alignItems="center" display="flex" pl={5}>
+            <CardActions style={{ paddingRight: 40 }}>
```
Comment (a31859) [inline lib/components/CTACard/CTACard.tsx:67]:
```suggestion
            <CardActions style={{ paddingRight: spacing(5) }}>
```
use theme spacing

**Review by a31859 — CHANGES_REQUESTED** [review]
Also use named exports. 
Instead of `export default MyComponent2` use `export const MyComponent2 = () => {}`

**Review by mstrk — APPROVED** [review]

**Pull request merged**